CONTRIBUTING.md

```diff
@@ -39,6 +39,13 @@ before being compiled into Python components that are in the
 `dash_bio/component_factory/` and must be imported in
 `dash_bio/__init__.py`.
 
+##### Installing new npm packages
+
+If developing a new component based on a React library, please ensure
+that you have already installed the correct versions of the
+preexisting dependencies by running `npm ci`. Then, add the package to
```

Comment:
Shouldn't we mention `rm -rf node_modules/` beforehand?

Reply:
Oh, I didn't realize that `npm ci` was taking care of it! :)
Reference: https://medium.com/better-programming/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26